docker/README.md

```diff
@@ -224,21 +225,49 @@ define( 'JETPACK__SANDBOX_DOMAIN', '{your sandbox}.wordpress.com' );
 
 ## Using Ngrok with Jetpack
 
-[Ngrok.com](https://ngrok.com/) free tier gives you one-use random-hash domains for HTTP/HTTPS connections and random ports for other TCP connections. When using their paid plans you can re-use domains, reserve your custom domains and reserve TCP ports.
+To be able to connect Jetpack you will need a domain - you can use [Ngrok.com](https://ngrok.com/) to assign one.
 
-If you use one-off domains, you'll have to re-install WordPress and re-connect Jetpack each time you close Ngrok (thus losing your randomly assigned domain). That's perfectly fine for quick testing or lightweight development.
+If you use one-off domains, you'll have to re-install WordPress and re-connect Jetpack each time you close Ngrok (thus losing your randomly assigned domain). That's perfectly fine for quick testing or lightweight development. You can use [other similar services](https://alternativeto.net/software/ngrok/) as well.
 
-You can use [other similar services](https://alternativeto.net/software/ngrok/) as well.
+If you're developing Jetpack often you'll want to reserve a domain you can keep using. If you create an account by signing up via Google and using your a8c Google account you'll be added to our business tier plan which will enable you to re-use domains, reserve your custom domains and reserve TCP ports.
 
-If you're developing Jetpack often with Ngrok, you could create a config file for your setup. See [default configuration file location](https://ngrok.com/docs#default-config-location) from Ngrok Docs or use `-config=your_config_file.yml` argument with `ngrok` to use your configuration file.
+Once you’ve joined the team, [reserve yourself a permanent domain here](https://dashboard.ngrok.com/reserved).
 
-In your configuration file, add:
+Once you’ve done that, follow [these steps](https://ngrok.com/download) to download and set up ngrok. However, instead of step four, edit your [config file](https://ngrok.com/docs#default-config-location) or use `-config=your_config_file.yml` argument with `ngrok` to use your configuration file.
```

Comment:
I would remove `or use `-config=your_config_file.yml` argument with `ngrok` to use your configuration file.` to keep things simple.

```suggestion
Once you’ve done that, follow [these steps](https://ngrok.com/download) to download and set up ngrok. However, instead of step four, edit your [config file](https://ngrok.com/docs#default-config-location) as explained below:
```